.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Why?

Reply:
for VS code, to give proper linting, checking, all of that stuff. We didn't have it, and after adding it, it showed like 50 errors.

Not sure if we want to keep it, I would say yes.

Reply:
Removed the file, it's VS code specific so let's not keep it.

Reply:
Personally, I like having things like this in the source code to help ensure consistency for multiple developers working on the code base.